one_fm/public/js/doctype_js/job_applicant.js

```diff
@@ -330,6 +330,14 @@ frappe.ui.form.on('Job Applicant', {
 			)
 		}
 	},
+	custom_transfer_reminder_date: function(frm){
+		var the_date = frm.doc.custom_transfer_reminder_date
+		if (the_date){
+			if (!(validateDate(the_date))){
+				frappe.throw("Oops! You can't choose a date in the past or today. Please select a future date.")
+			}
```

Comment:
You can simply check
```
if(frm.doc. custom_transfer_reminder_date < frappe.datetime.now_date()){
  frappe.throw("Oops! You can't choose a date in the past or today. Please select a future date.")
}
```

Ref: https://github.com/ONE-F-M/One-FM/blob/f359ad66ae800e1076aa860d47a54f68e2432108/one_fm/one_fm/doctype/erf/erf.js#L776